src/benchmark.cc

```diff
@@ -877,8 +881,8 @@ void ParseCommandLineFlags(int* argc, const char** argv) {
                        &FLAGS_benchmark_repetitions) ||
         ParseStringFlag(argv[i], "benchmark_format",
                         &FLAGS_benchmark_format) ||
-        ParseBoolFlag(argv[i], "color_print",
-                       &FLAGS_color_print) ||
+        ParseStringFlag(argv[i], "color_print",
```

Comment:
How lenient should the check be? ParseBoolFlag() is quite generous with what it accepts, as is Google Test's --gtest_color option. 

I've changed it to assert that the user passes a non-empty string. Is that enough?